lib/tasks/knows_what_exercises_are_called_and_where_they_live_in_the_directory_structure.rb

```diff
@@ -0,0 +1,34 @@
+class Exercise
+  def self.all
+    FileList['exercises/*'].pathmap('%f').map { |e| new(e) }
```

Comment:
If the exercise abstraction is a good one, should this return instances of exercises, or exercise names (and would be turned into exercise instances currently in the Rakefile)?